tests/modules/programs/dircolors/settings.nix

```diff
@@ -0,0 +1,162 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    programs.dircolors = {
+      enable = true;
+
+      settings = {
+        OTHER_WRITABLE = "30;46";
+        ".sh" = "01;32";
+        ".csh" = "01;32";
+      };
+
+      extraConfig = ''
+        # Extra dircolors configuration.
+      '';
+    };
+
+    nmt.script = ''
+      assertFileContent \
+        home-files/.dir_colors \
+        ${
+          pkgs.writeText "dircolors-expected-config.conf" ''
```

Comment:
why not give it a static .conf in the same dir instead of writing one from nix?

Reply:
I think the code is easier to read if the whole test is in one file. A separate conf file for each test would get especially unwieldy if there are multiple tests.

Reply:
I can agree that it is nice to keep in a single file but in this case the text is very long and I would suggest following @kalbasit's suggestion and putting it in a separate `settings-expected.conf` file and just referring to it as `${./settings-expected.conf}` here.

Reply:
Done.